src/js/ctrl/export.ts

```diff
@@ -105,7 +110,7 @@ exporter.widgets = function(state: State, spec) {
 }
 
 exporter.pipelines = function(state: State, internal: boolean) {
-  const pipelines: PipelineRecord[] = getInVis(state, 'pipelines').valueSeq();
+  const pipelines: PipelineRecord[] = state.getIn(['pipelines']).valueSeq();
```

Comment:
this is the right idea. however, you've changed the type of `state` from `State` to `VisStateTree` and should update type annotations to match